charts/orchestrator/values.yaml

```diff
@@ -121,6 +121,8 @@ backstage:
               target: https://github.com/parodos-dev/workflow-software-templates/blob/main/entities/workflow-resources.yaml
             - type: url
               target: https://github.com/parodos-dev/workflow-software-templates/blob/main/template/template.yaml
```

Comment:
@masayag this won't work anymore with the [new structure](https://github.com/parodos-dev/workflow-software-templates/tree/main/template) just merged. This line should be changed to https://github.com/parodos-dev/workflow-software-templates/blob/main/template/infrastructure/template.yaml
As we discussed this morning, we can either keep as-is if it won't break anything for now and I will restructure the infrastructure template to be generic and serve both assessment and infrastructure template or change the line.
cc @dmartinol

Reply:
Good catch.